assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -382,6 +386,11 @@
             <artifactId>tomcat-catalina</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-impl</artifactId>
+            <scope>runtime</scope>
```

Comment:
why did you added `  <scope>runtime</scope>` ?

Reply:
I would add this dependency to exclusion https://github.com/eclipse-che/che-server/blob/672ae77018ace080da9e388ba4a3491e910bcde6/assembly/assembly-wsmaster-war/pom.xml#L435-L457

Reply:
fixed